### DISCUSSION TITLE
DoubleTapZoom

### DISCUSSION THREAD

**Discussion by ArthurZuliani** [stephanrauh/ngx-extended-pdf-viewer#1476] [Q&A]
I am wondering if there is a way to disable double tap zoom?

Thanks

**Comment by stephanrauh** [comment]
Yes. For example, you can add an event listener to `<ngx-extended-pdf-viewer>` or - even better - one of it's internal `<div>` tags and prevent the double click. For example, see https://github.com/stephanrauh/ngx-extended-pdf-viewer/issues/970.
